tests/test_other.py

```diff
@@ -8053,8 +8053,8 @@ def test(args, closure, opt):
 
   # TODO: Debug why the code size is different on Windows and Mac. Also, for
   # some unknown reason (at time of writing), this test is not skipped on the
-  # Windows autoroller, despite the bot being correctly configured to skip this
-  # test. The no_windows decorator also solves that problem.
+  # autoroller, despite the bot being correctly configured to skip this test.
```

Comment:
It might be worth noting that the skipping works on Linux but not on MacOS or Windows.

Reply:
Done